test/sharness/t0100-name.sh

```diff
@@ -99,6 +99,12 @@ test_expect_success "'ipfs name publish' fails" '
   printf '' | test_expect_code 1 ipfs name publish >publish_out 2>&1
 '
 
+test_expect_success "'ipfs name publish --help' succeeds" '
+  PEERID=`ipfs id --format="<id>"` &&
```

Comment:
whats with the ipfs id check?

Reply:
Copied from the other tests. I have no idea but I assume it's to make sure that publishing would otherwise work (although I guess it shouldn't really be necessary in this case).

Reply:
(fixed, and put the tests in the correct order)